lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -5,45 +5,46 @@ defmodule ExUnit.Runner do
 
   @rand_algorithm :exs1024
 
-  def run(opts, load_us) do
-    {:ok, manager} = EM.start_link()
-    {:ok, stats} = EM.add_handler(manager, ExUnit.RunnerStats, opts)
-    {opts, config} = configure(manager, opts)
+  @type manager :: ExUnit.EventManager.manager()
```

Comment:
We can't do this because the type is privat. Are we even using it?

Reply:
I have the feeling I do in the main PR.
I will remove it for now and add it back if needed

Reply:
It shouldn't be necessary in the main PR either because we don't use spec for private functions or private functionality and everything here is private.